packages/core/assistant-testing/src/blueprints/design/design.test.ts

```diff
@@ -94,8 +95,8 @@ describe.runIf(process.env.DX_RUN_SLOW_TESTS === '1')('Design Blueprint', { time
       Effect.provide(
         Layer.mergeAll(
           TestDatabaseLayer({ types: [DataType.Text, Markdown.Document, Blueprint.Blueprint] }),
-          makeToolResolverFromFunctions([readDocument, updateDocument]),
-          makeToolExecutionServiceFromFunctions([readDocument, updateDocument]),
+          makeToolResolverFromFunctions([readDocument, updateDocument], testToolkit),
```

Comment:
what is this for?

Reply:
what do you mean? we updated this to take a toolkit so its passing in a toolkit